spec/sequel/plugins/enum_values_spec.rb

```diff
@@ -229,6 +229,62 @@
 			end
 		end
 	end
+
+	describe 'Sequel::Model.enum_filter' do
```

Comment:
There is no such method. You can write regular text, like `describe 'dataset filters by enum value'`.

Also now I see that this is should be inside `describe 'Sequel::Model.enum_values'`, after `describe 'predicate_methods option'`.

Reply:
replaced it